config.toml

```diff
@@ -35,14 +35,12 @@ enableEmoji = true
 blog = "/:section/:year/:month/:day/:slug/"
 
 [markup]
-defaultMarkdownHandler = "blackfriday"
+defaultMarkdownHandler = "goldmark"
 
 ## Configuration for BlackFriday markdown parser: https://github.com/russross/blackfriday
```

Comment:
One last thing, please update this comment so the description matches what the config below does.

Reply:
@geriom , changed to goldmark reference link, good for re-review